roles/engineer.md

```diff
@@ -12,37 +12,14 @@ We need someone:
 
 We expect someone in the role:
 
+- to be billable for at least half their time after 2 weeks on-boarding
```

Comment:
Yes I think on project work we can probably say most people will be fully
billable. It's well known that Support isn't at such utilisation, not just
down to experience level but the bitty type of work that can sometimes
occur. Of course as we get better at using iterations to ship Support work
then I guess we will see utilisation go up too.

On Tue, Aug 16, 2016 at 4:30 PM, Chris Blackburn <notifications@github.com>
wrote:

> In roles/engineer.md
> <https://github.com/madetech/handbook/pull/15#discussion_r74958762>:
>
> > @@ -12,37 +12,14 @@ We need someone:
> >
> >  We expect someone in the role:
> >
> > +- to be billable for at least half their time after 2 weeks on-boarding
>
> The current way we bill AaaS teams requires a full-time ("good faith")
> billing commitment. Not to say that's set in stone necessarily, but would
> require a billing rethink.
>
> Given where we're currently hiring, do we have an expectation that
> everyone joining the team could be fully billable in two weeks? I think the
> difference was when we were focusing our hiring around less experienced
> engineers?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/madetech/handbook/pull/15/files/1f0ae1a8335aad96a38bb0d2aa349d6c182a7cd7#r74958762>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAQpADXOW-QKOqI3-g_9Y4RymBJ5VZfqks5qgdeOgaJpZM4JlhI3>
> .
>